lib/pkgcloud/openstack/cdn/client/base.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * flavors.js: Instance methods for working with base resources from Openstack CDN
+ *
+ * (C) 2014 Rackspace
+ *      Shaunak Kashyap
+ * MIT LICENSE
+ */
+
+/**
+ * client.getHomeDocument
+ *
+ * @description gets the home document for the CDN service
+ *
+ * @param callback
+ * @return {*}
+ */
+exports.getHomeDocument = function(callback) {
+  var requestOptions = {
+    path: '/'
+  };
+
+  return this._request(requestOptions, function (err, body) {
+    if (err) {
+      callback(err);
+      return;
+    }
+
+    callback(null, body);
+  });
+};
+
+/**
+ * client.getPing
+ *
+ * @description gets the server ping response (status response)
+ *
+ * @param callback
+ * @return {*}
+ */
+exports.getPing = function(callback) {
+  var requestOptions = {
+    path: '/ping'
+  };
+
+  return this._request(requestOptions, function (err) {
+    if (err) {
```

Comment:
Again, can simply invoke `callback(err);` inside the handler. 

One reason we don't simply pass callback to `this._request` is to not return raw data from the request handler.